server/server-config.yaml

```diff
@@ -0,0 +1,26 @@
+host: http://localhost:8080
```

Comment:
we need to add documentation in config file

Reply:
gonna add in the forthcoming PR, this one has grew too large :)